### PR TITLE
Turn on approvers for k/kubernetes-template-project

### PR DESCRIPTION
**Pull request by spiffxp**

### CODE REVIEW

File: prow/plugins.yaml [hunk 1]
```diff
@@ -126,6 +126,9 @@ plugins:
   - docs-no-retest
   - blockade
 
+  kubernetes/kubernetes-template-project:
+  - approve
```
Comment (cjwagner) [inline prow/plugins.yaml]: The approve plugin config defaults to no associated issue required and no implicit author self-approval. Are those defaults what you want?

Comment (spiffxp) [reply prow/plugins.yaml]: yes I'm fine with that

**Comment by BenTheElder** [comment]
@spiffxp merge whenever the project is ready :-)

**Pull request merged**